pycbc/strain.py

```diff
@@ -354,8 +381,11 @@ def from_cli(opt, dyn_range_fac=1, precision='single'):
                              pd_taper_window) )
         gate_data(strain, gate_params)
 
-
+    strain.injcutter = injcutter
```

Comment:
Ok, all the stuff below seems to add the injection rejection options to the strain group. This should be a separate group, and the options should be passed to the injection rejection directly in pycbc_inspiral.